recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 2
```

Comment:
I suspect this is `2` becuase of #62, in which case we'll probably want to wait to merge that before we merge this.

Reply:
That's correct.